lib/ui/desc_persona.dart

```diff
@@ -33,18 +37,157 @@ class _DescPersonaScreenState extends State<DescPersonaScreen> {
     Get.offNamed(ShowNewsScreen.route);
   }
 
+  progressBarTimer() async {
+    timer = Timer.periodic(
+        const Duration(milliseconds: 20), (_) => setProgressBarValue());
+  }
+
+  setProgressBarValue() {
+    setState(() {
+      if (barValue < 1) {
+        barValue = barValue + 0.01;
+      } else {
+        timer?.cancel();
```

Comment:
It seems the timer is not cancelled as it should be. After the screen transitions away from `desc_persona` an exception occurs:
```
/flutter (10489): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: setState() called after dispose(): _DescPersonaScreenState#556a7(lifecycle state: defunct, not mounted)
E/flutter (10489): This error happens if you call setState() on a State object for a widget that no longer appears in the widget tree (e.g., whose parent widget no longer includes the widget in its build). This error can occur when code calls setState() from a timer or an animation callback.
E/flutter (10489): The preferred solution is to cancel the timer or stop listening to the animation in the dispose() callback. Another solution is to check the "mounted" property of this object before calling setState() to ensure the object is still in the tree.
E/flutter (10489): This error might indicate a memory leak if setState() is being called because another object is retaining a reference to this State object after it has been removed from the tree. To avoid memory leaks, consider breaking the reference to this object during dispose().
E/flutter (10489): #0      State.setState.<anonymous closure> (package:flutter/src/widgets/framework.dart:1052:9)
E/flutter (10489): #1      State.setState (package:flutter/src/widgets/framework.dart:1087:6)
E/flutter (10489): #2      _DescPersonaScreenState.setProgressBarValue (package:have_you_heard/ui/desc_persona.dart:47:5)
E/flutter (10489): #3      _DescPersonaScreenState.progressBarTimer.<anonymous closure> (package:have_you_heard/ui/desc_persona.dart:43:50)
E/flutter (10489): #4      _rootRunUnary (dart:async/zone.dart:1436:47)
E/flutter (10489): #5      _CustomZone.runUnary (dart:async/zone.dart:1335:19)
E/flutter (10489): #6      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1244:7)
E/flutter (10489): #7      _CustomZone.bindUnaryCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1281:26)
E/flutter (10489): #8      _rootRunUnary (dart:async/zone.dart:1444:13)
E/flutter (10489): #9      _CustomZone.runUnary (dart:async/zone.dart:1335:19)
E/flutter (10489): #10     _CustomZone.bindUnaryCallback.<anonymous closure> (dart:async/zone.dart:1265:26)
E/flutter (10489): #11     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:395:19)
E/flutter (10489): #12     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:426:5)
E/flutter (10489): #13     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
E/flutter (10489): 
```